### ISSUE TITLE
Support for Debian Linux

### ISSUE THREAD

**Issue by actraiser** [mbrother2/backuptogoogle#6]
##### Detail log:
```
[ 25/11/2019 09:04:00 ] Directory /root/bin existed. Skip
[ 25/11/2019 09:04:00 ] Check write to /root/bin successful
[ 25/11/2019 09:04:00 ] Cheking network...
[ 25/11/2019 09:04:00 ] Connect Github successful
[ 25/11/2019 09:04:00 ] Connect Google successful
[ 25/11/2019 09:04:00 ] Checking OS...
[ 25/11/2019 09:04:00 ] Sorry! We do not support your OS. Exit
```

Could you kindly support Debian Linux? Right now, the above message is shown when running butgg.

Greets
-act

**Comment by mbrother2** [comment]
@actraiser 

I will test my script on Debian. If everything is ok, I will add support Debian.

**Comment by mbrother2** [comment]
@actraiser 

I tested my script on Debian. No issue is detected. Please try! Remember download right script for Linux:
```
curl -o butgg.bash https://raw.githubusercontent.com/mbrother2/backuptogoogle/master/butgg.bash
bash butgg.bash --setup
```

**Comment by mbrother2** [comment]
@actraiser 

Did you try? Is everything ok?

**Comment by actraiser** [comment]
Many thanks @mbrother2! With your correct command I was able to trigger the build process. There were a couple of things I had to do along the way though.

First of all, my Go Version was too new and the script exited with the following error.

```
It compile: version "go1.13.1" does not match go tool version "go1.12.5"
```

So I installed Go 1.12.5 as described on [this page](https://www.digitalocean.com/community/tutorials/how-to-install-go-on-debian-9)

Now the setup ran through but the 'gdrive about' command returned a 401 error, so I had to finally run the following command to authorize my gdrive binary.

```
bash butgg.bash --setup credential
```

After that, everything worked! Thanks a lot for the script!

Greets
-act
